application/controllers/UserManagementController.php

```diff
@@ -1163,9 +1164,8 @@ public function actionBatchApplyRoles()
             $aResults[$sItem]['title'] = '';
             $model = $this->loadModel($sItem);
             $aResults[$sItem]['title'] = $model->users_name;
-
-            //check if user is admin otherwhise change the role
-            if (intval($sItem) == 1) {  //todo REFACTORING is admin id always 1?? is there another possibility to check for admin user?
+            if (Permission::isForcedSuperAdmin($sItem)) {
```

Comment:
Same comment as in other PR.
Not sure this is part of the ticket. Must be clarified for better testing.

Reply:
It's clear the `//todo REFACTORING is admin id always 1??`

Permission::isForcedSuperAdmin id done to replace loginid == 1 …

Reply:
https://github.com/LimeSurvey/LimeSurvey/pull/810

Reply:
> It's clear the //todo REFACTORING is admin id always 1??

It is clear in the PR, but will the tester know that? Is it on the ticket?